### DISCUSSION TITLE
Are aliases supported

### DISCUSSION THREAD

**Discussion by simonhanmer** [Lednerb/bilberry-hugo-theme#411] [Q&A]
Hi there,
 just trying out the theme - I've setup a test site and copied over some content from my main site.
 
 In one of my posts, I've got an aliases section in the frontmatter as below:
```
aliases:
  - /alternate_url/
```

but the alias isn't used, it's just picking up the url from the page bundle path.

**Comment by igor-baiborodine** [comment]
@simonhanmer, it's unclear what you are trying to achieve with aliasing in your case. The `alternate_url` value should be an old relative URL under which this article was published previously on your Hugo-based website. For more details, you can read this [post](https://www.kiroule.com/article/use-aliases-to-redirect-old-urls/).
